Cargo.toml

```diff
@@ -16,7 +16,7 @@ libc = ">=0.2.39"
 serde = { version = ">=1.0.27", optional = true }
 serde_derive = { version = ">=1.0.27", optional = true }
 
-[target.'cfg(unix)'.dependencies]
+[target.'cfg(linux)'.dependencies]
```

Comment:
This should be `[target.'cfg(target_os = "linux")'.dependencies]`.

`cfg(linux)` is not defined (just unix is :))) ). I just noticed that because the coverage is increased (even though it shouldn't be). Every occurrence of `#[cfg(linux)]` should be replaced with `#[cfg(target_os = "linux")]`.

Reply:
Updated, thanks!